main.js

```diff
@@ -211,89 +222,110 @@ function createPost() {
         }
         
     };
-const borderPhone = document.createElement('div');
-borderPhone.className = "phone";
-    document.body.append(borderPhone)
 
+    //Creating container 1
     const cnt = document.createElement('div');
-        cnt.className = "container";
-        borderPhone.append(cnt) 
-
-            const avatar = document.createElement('div')
-                avatar.className = "avatar_container";
-                cnt.append(avatar)
-                    const avatarImg = document.createElement('img')
-                        avatarImg.className = "avatar border";
-                        avatarImg.src = "./avatar.jpg";
-                        avatarImg.alt = "";
-                        avatar.append(avatarImg)
-                const headerTexts = document.createElement('div')
-                    headerTexts.className = "header_texts";
-                    avatar.append(headerTexts)
-                        const nameUser = document.createElement('div')
-                            nameUser.className = "name";
-                            nameUser.textContent = "Name";
-                            headerTexts.append(nameUser)
-                        const songUser = document.createElement('div')
-                            songUser.className = "song";
-                            songUser.textContent = "S.O.A.D. - Toxicity";
-                            headerTexts.append(songUser) 
+    cnt.className = "container";
+    cnt.style.backgroundImage = backgroundImg;
+    borderPhone.append(cnt) 
+
+    //creating avatar container
+    const avatar = document.createElement('div')
+    avatar.className = "avatar_container";
```

Comment:
as mentioned above, these line repeat very often, fn is needed to write the logic only once. Parameters will help to create elements with different tags and give it a custom class

[read this doc](https://learn.javascript.ru/function-basics) and writing each fn from it will give you more understanding how functions work